### PR TITLE
Add blockbench icon

### LINKED ISSUE #8041: Blockbench Icon
### Brand Name

Blockbench

### Website

https://www.blockbench.net/

### Popularity Metric

Similarweb rank: # 85,086 see [https://www.similarweb.com/it/website/blockbench.net/](https://www.similarweb.com/it/website/blockbench.net/)

### Official Resources for Icon and Color

All logos, colors and info: [https://www.blockbench.net/wiki/blockbench/logos](https://www.blockbench.net/wiki/blockbench/logos)

Main logo: [https://www.blockbench.net/images/logos/icon.png](https://www.blockbench.net/images/logos/icon.png)
White logo: [https://www.blockbench.net/images/logos/blockbench_inverted.png](https://www.blockbench.net/images/logos/blockbench_inverted.png)
Logo with text: [https://www.blockbench.net/images/logos/blockbench_logo_text.png](https://www.blockbench.net/images/logos/blockbench_logo_text.png)

SVG
Blockbench logo

Color: Placid Blue `#1e93d9`
Second Color: Plain White `#ffffff`
Background color: Deep Ocean Blue `#212e3c`

### Additional Comments

There are two variant, I think the "White Logo" is the best for this purpose

### Contributing

Yes

### PR DESCRIPTION
**Pull request by Fabrimat**
blockbench

**Issue:** closes #8041 

**Similarweb rank:**
Rank # 85,086 see https://www.similarweb.com/it/website/blockbench.net/

### Checklist
  - [x] I updated the JSON data in `_data/simple-icons.json`
  - [x] I optimized the icon with SVGO or SVGOMG
  - [x] The SVG `viewbox` is `0 0 24 24`

### CODE REVIEW

**Review by adamrusted — CHANGES_REQUESTED** [review]
I'm seeing [some differences](https://github.com/simple-icons/simple-icons/assets/1830867/a4b4f4fb-9d6a-4d14-a206-650e4e71e9d2) when comparing this PR (black) against `blockbench_logo_black.svg` available in their logo pack. Would you mind taking another look at this @Fabrimat? Just for reference, the JSON will now fail the linter if you leave the trailing forward-slashes in there too.

File: _data/simple-icons.json [hunk 1]
```diff
@@ -1962,6 +1962,12 @@
             "source": "https://www.blender.org/about/logo",
             "guidelines": "https://www.blender.org/about/logo"
         },
+        {
+            "title": "Blockbench",
+            "hex": "1E93D9",
+            "source": "https://www.blockbench.net/wiki/blockbench/logos/",
```
Comment (PeterShaggyNoble) [inline _data/simple-icons.json:1968]:
```suggestion
            "source": "https://www.blockbench.net/wiki/blockbench/logos",
```

File: _data/simple-icons.json [hunk 2]
```diff
@@ -1962,6 +1962,12 @@
...
+            "guidelines": "https://www.blockbench.net/wiki/blockbench/logos/#usage-guidelines"
```
Comment (PeterShaggyNoble) [inline _data/simple-icons.json:1969]:
```suggestion
            "guidelines": "https://www.blockbench.net/wiki/blockbench/logos"
```

File: icons/blockbench.svg [hunk 3]
```diff
@@ -0,0 +1 @@
....5c-.25 0-1 .06-3.47.22l-9.11.61c-3.05.2-2.75.18-2.89.26-.16.1-.25.27-.3.57-.46 2.12-.82 3.64-1.11 4.88-.02.04-.05.17-.07.3-.04.2-.04.23.01.3.1.14.23.17 1.14.28l10.21 1.31 3.2.41c.93.12.95.12 1 .02l.2-.94 1.43-7.23c.13-.63.13-.7.07-.8a.5.5 0 0 0-.15-.15c-.04-.02 0-.04-.16-.04zM6 11.3a2.11 2.06 0 0 0-.35.03l-2.79.39c-1.37.19-2.55.36-2.62.39-.21.07-.3.27-.2.46.06.12.22.19.7.32l.2.05c4.53 1.24 9.33 2.61 13.58 3.8.23.06.56.18.77.18a175.29 175.29 0 0 0 6.13-2.98c.11-.1.17-.3.11-.39a1.34 1.34 0 0 0-.36-.08 1433.84 1433.84 0 0 0-4.08-.59 1248.36 1216.4 0 0 1-1.61-.22l-1.4-.2a1189.81 1159.35 0 0 1-3.29-.47 488.81 488.81 0 0 0-4.1-.6A3.4 3.4 0 0 0 6 11.3Zm-1.68 3.18a.06.06 0 0 0-.02 0c-.05.01-1.12 2.25-1.12 2.35 0 .1.08.23.17.28.1.05.17.05.31 0l1.36-.41c1.32-.39 1.34-.4 1.48-.54.11-.11.33-.54.39-.79a.81.79 0 0 1 .06-.15l.02-.06a168.19 163.88 0 0 1-2.3-.6l-.35-.08zm13.64 1.75a44.09 44.09 0 0 1-.57.3c-.48.28-.94.58-1.44.8-.42.26-.6.32-.8.25-.2-.06-.74-.2-.79-.2a.08.07 0 0 0-.07.05l-.19.52c-.2.5-.37 1-.55 1.48-.08.23-.23.75-.23.81 0 .06.14.2.25.23a.55.54 0 0 0 .23.03c1.26-.5 2.21-1.12 3.28-1.67.33-.17.45-.28.5-.48a62.53 62.53 0 0 0 .27-1.17c.18-.79.19-.86.14-.93 0-.02-.01-.02-.03-.02z"/></svg>
```
Comment (PeterShaggyNoble) [inline icons/blockbench.svg:1]: As this PR has been stale for a while, I recreated this myself from `blockbench_logo_white.svg` found in the [zip file](https://www.blockbench.net/download_files/blockbench_logo_kit_v1.zip) at [source](https://www.blockbench.net/wiki/blockbench/logos/). I tried to remove as many excess points as I could while maintaining the integrity of the icon but someone may be able to improve further by taking another pass at it/
```suggestion
<svg role="img" viewBox="0 0 24 24" xmlns="http://www.w3.org/2000/svg"><title>Blockbench</title><path d="M23.651 3.277c-.25.002-1.009.057-3.47.226l-9.113.627c-3.044.208-2.743.18-2.888.27-.163.102-.254.274-.308.58-.451 2.177-.812 3.74-1.104 5.007-.015.04-.046.178-.07.306-.038.218-.037.239.012.307.105.144.23.176 1.14.294 4.78.63 8.905 1.164 13.412 1.765.931.124.948.125.99.022.032-.081.091-.36.207-.971.476-2.468 1.03-5.366 1.433-7.42.126-.642.132-.712.069-.82a.507.507 0 0 0-.153-.153c-.04-.024-.007-.042-.157-.04zM5.65 11.31c-.161.025-1.416.205-2.79.4-1.373.194-2.554.374-2.624.399-.21.076-.294.285-.194.478.062.12.215.19.7.324.065.018.15.043.19.056 4.54 1.274 9.338 2.68 13.583 3.888.238.071.563.197.77.197.157-.042 1.053-.484 3.082-1.505 1.649-.83 3.022-1.53 3.052-1.555.113-.099.173-.31.113-.402-.012-.018-.173-.055-.357-.082-3.949-.579-8.122-1.19-11.684-1.71-2.014-.29-2.635-.384-2.804-.422a3.329 3.329 0 0 0-.647-.09 2.254 2.254 0 0 0-.39.024zM4.32 14.55h-.016c-.054.011-1.122 2.303-1.122 2.407 0 .107.076.238.168.287.101.055.165.056.312.005.445-.139.918-.293 1.36-.425 1.316-.395 1.337-.403 1.477-.549.115-.12.328-.562.39-.811a.803.803 0 0 1 .057-.154c.02-.035.03-.065.024-.065-.768-.198-1.565-.415-2.292-.614a2.947 2.947 0 0 0-.358-.081zm13.52 1.843c-.07.04-.271.155-.448.254-.481.291-.937.593-1.446.833-.413.259-.594.315-.801.248a9.005 9.005 0 0 0-.786-.204.076.076 0 0 0-.067.05c-.23.643-.507 1.395-.74 2.053-.086.232-.236.767-.234.832.002.064.142.196.25.236a.551.551 0 0 0 .231.027c1.257-.51 2.215-1.15 3.284-1.71.327-.171.447-.29.494-.492.094-.407.182-.808.269-1.2.183-.812.194-.888.148-.961a.038.038 0 0 0-.035-.02c-.054.012-.08.036-.118.054z"/></svg>
```

**Review by PeterShaggyNoble — CHANGES_REQUESTED** [review]

**Pull request merged**